Belfast.md

```diff
@@ -0,0 +1,5 @@
+# Belfast, UK
+
+### Places to eat
+
+### Things to do.
```

Comment:
I imagine that visiting shooting locations for Game of Thrones is a popular activity.